core/domain/suggestion_services_test.py

```diff
@@ -2084,6 +2094,1155 @@ def test_create_translation_contribution_stats_from_model(self) -> None:
+        # to create/update translation contribution stats.
+        assert translation_contribution_stats_model is not None
+        self.assertEqual(
+            translation_contribution_stats_model.submitted_translations_count,
+            2
+        )
+        self.assertEqual(
+            (
+                translation_contribution_stats_model
+                .submitted_translation_word_count
+            ),
+            6
+        )
+        self.assertEqual(
+            translation_contribution_stats_model.accepted_translations_count,
+            0
+        )
+
+    def test_update_translation_review_stats_when_suggestion_is_accepted(
+        self) -> None:
+        # This test case will check stats of the reviewer and the submitter
+        # when a translation suggestion is accepted.
+        # Steps required in the setup phase before testing.
+        # 1. Create and publish explorations.
+        # 2. Create and publish topics.
+        # 3. Create stories for translation opportunities.
+        # 4. Save translation suggestions.
+        change_dict = self._set_up_topics_and_stories_for_translations()
+        initial_suggestion = suggestion_services.create_suggestion(
+            feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            feconf.ENTITY_TYPE_EXPLORATION,
+            '0', 1, self.author_id, change_dict, 'description')
+        latest_suggestion = suggestion_services.create_suggestion(
+            feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            feconf.ENTITY_TYPE_EXPLORATION,
+            '1', 1, self.author_id, change_dict, 'description')
+
+        suggestion_services.update_translation_contribution_stats_at_submission(
+            initial_suggestion
+        )
+        suggestion_services.update_translation_contribution_stats_at_submission(
+            latest_suggestion
+        )
+        translation_contribution_stats_model = (
+            suggestion_models.TranslationContributionStatsModel.get(
+                'hi', self.author_id, '0'
+            )
+        )
+        # Assert translation contribution stats before the review.
+        # At this point we can confirm that there should be an associated
+        # translation contribution stat object for the given IDs since we have
+        # called update_translation_contribution_stats_at_submission function
+        # to create/update translation contribution stats.
+        assert translation_contribution_stats_model is not None
+        self.assertEqual(
+            translation_contribution_stats_model.submitted_translations_count,
+            2
+        )
+        self.assertEqual(
+            (
+                translation_contribution_stats_model
+                .submitted_translation_word_count
+            ),
+            6
+        )
+        self.assertEqual(
+            translation_contribution_stats_model.accepted_translations_count,
+            0
+        )
```

Comment:
Since we've already tested this above, I would just remove these lines here and in the tests below.

Reply:
Tt is also added as a base verification of contributor stats before it is reviewed.